interview-questions.md

```diff
@@ -8,51 +8,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. What are props in React?
 
   Your answer:
+  React props are properties. They can only be passed one way and are read-only and immutable.
 
   Researched answer:
-
+  To keep a component's inherited data dynamic with the JSX file, props helps pass data from the JSX file down to a component as a parameter.
 
 
 2. What is a DOM event?
 
   Your answer:
+  DOM events are the refreshing of nodes upon user interaction that is then displayed by use of the much faster Virtual DOM.
 
   Researched answer:
-
+  Better put, the events are all of the different ways developer can control how the VDOM updates and isolates the different event listeners, only updating the page at their location. The different user interaction categories that can be accounted for are UiEvent, Event, AnimationEvent, FocusEvent, MouseEvent, ClipboardEvent, DragEvent, ProgressEvent, HashChangeEvent, InputEvent, KeyboardEvent, WheelEvent, PageTransitionEvent, PopStateEvent, StorageEvent, TouchEvent, and TransitionEvent. All of which have different event listeners related to their categories.
```

Comment:
The DOM events are very attentive to every key stoke of the user. Each event are signals or triggers to state that something is occurring or has occurred.